qiskit/visualization/state_visualization.py

```diff
@@ -807,21 +833,30 @@ def plot_state_qsphere(rho, figsize=None, ax=None):
     ax2 = fig.add_subplot(gs[2:, 2:])
     ax2.pie(theta, colors=sns.color_palette("hls", n), radius=0.75)
     ax2.add_artist(Circle((0, 0), 0.5, color='white', zorder=1))
-    ax2.text(0, 0, 'Phase', horizontalalignment='center',
-             verticalalignment='center', fontsize=14)
-
     offset = 0.95  # since radius of sphere is one.
 
-    ax2.text(offset, 0, r'$0$', horizontalalignment='center',
-             verticalalignment='center', fontsize=14)
-    ax2.text(0, offset, r'$\pi/2$', horizontalalignment='center',
-             verticalalignment='center', fontsize=14)
-
-    ax2.text(-offset, 0, r'$\pi$', horizontalalignment='center',
-             verticalalignment='center', fontsize=14)
-
-    ax2.text(0, -offset, r'$3\pi/2$', horizontalalignment='center',
-             verticalalignment='center', fontsize=14)
+    if use_degrees:
```

Comment:
Could you change this so that there isn't so much duplicated code? Maybe something like
``` 
if use_degrees :
     labels = ['0', '90', '180', '270']
else : 
      labels = ['$0$', '$\pi/2$', '$\pi$', '$3\p/2i$']

# then the ax.text code with these labels
```

Reply:
done -- only one set of `ax.text` code now